python/requirements-dev.txt

```diff
@@ -0,0 +1,18 @@
+# This file was autogenerated by uv via the following command:
+#    uv pip compile python/requirements-dev.in
```

Comment:
What do you think about using the new `uv` commands that maintain a `uv.lock` instead of using pip-compile?